DMCompiler/Compiler/DM/DMASTHelper.cs

```diff
@@ -7,11 +7,12 @@
 using OpenDreamShared.Dream;
 using OpenDreamShared.Dream.Procs;
 using OpenDreamShared.Compiler;
+using Robust.Shared.Analyzers;
```

Comment:
This should be using the VirtualAttribute from `DMCompiler` not `Robust.Shared.Analyzers`